src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -54,6 +58,20 @@ export type OpeningPeriodFormProps = {
   errorTextAndLabel: NotificationTexts;
 };
 
+const validateStartInputWithEndDate = (end: Date | null) => (
+  start: string | null
+): boolean | string => {
```

Comment:
Is this how the react-hook-form validation works? So that if you return a boolean value true, it is valid, but if you return anything else than exactly that (for example a text string), it interpretes it as invalid?

Reply:
Yes, this is how it seems to work:
- true === valid
- (false || string) === invalid

Here is an example code of custom validation: https://github.com/react-hook-form/react-hook-form/blob/master/examples/customValidation.tsx